content/docs/sidebar-toolbar.md

```diff
@@ -80,15 +80,15 @@ interface ToolbarOptions {
 }
 ```
 
-| key         | usage                                                                                                                                                          |
-| ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| key | usage |
+| --- | --- |
 | **buttons** | _Deprecated — [Configure on the form instead](/docs/forms#customizing-form-buttons)_: Customizes the string displayed on either the 'save' or 'reset' buttons. |
 
 ![tina toolbar ui](/img/toolbar-ex.jpg)
 
 On its own, the toolbar will display the 'Save' and 'Reset' buttons, along with a form status indicator to show whether there are unsaved changes. [Custom Widgets](/guides/nextjs/github/toolbar-plugins) can also be added to extend functionality for the particular workflow.
 
-> _Note:_ **It is now recommended to configure the 'Save' & 'Reset' button text [on the form](/docs/forms#customizing-form-buttons)** intead of in the UI options. Please note that if `buttons` are configured on the CMS through the `sidebar` or `toolbar` options (as in the examples below), those values will take precedent over custom button values passed to a form.
+> _Note:_ **It is now recommended to configure the 'Save' & 'Reset' button text** [**on the form**](/docs/forms#customizing-form-buttons) intead of in the UI options. Please note that if `buttons` are configured on the CMS through the `sidebar` or `toolbar` options (as in the examples below), those values will take precedence over custom button values passed to a form.
```

Comment:
To start off, thanks so much for the grammar lesson; being a diverse team spread across two countries, I'm sure this isn't the only misuse of a word in our documentation.

That said, I'm concerned with the _other_ changes in this PR that I think are due to your `prettier` config clashing with ours.  Most of these seem harmless (swapping `-` for `*`), but later ones (line 121) have me thinking a smaller PR with _just_ the text change might be best.

Reply:
Hi,  yes I agree and think the only change you need is that precedent -> precedence change.

This might be more  feedback for your product then...   All I did was click the TinaCMS editor, changed that one word (precedent -> precedence) and clicked save.  Any other changes you see on there must have been made by your tools converting to markdown, etc.